javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,142 @@
+//voice assistent program
+let myname = "";
+const toDoList = [];
+
+function getReply(command) 
+{
```

Comment:
this function is really long. As a rule of thumb functions should never exceed the length of 5-15 lines of code.

However i understand why you would put it all together as this is naturally connected. However the functionality of the code might be clearer to an outsider if you create a function for each chatbot feature, ie. 'addTodo(todoName)' etc.

Reply:
Here i made separate function depending on functionality.

Reply:
if every thing looks ok. May i push my code in master.